setup/command/apply.go

```diff
@@ -3,7 +3,7 @@ package command
 import (
 	"fmt"
 
-	"github.com/quintilesims/layer0/setup/instance"
+	setup_instance "github.com/quintilesims/layer0/setup/instance"
```

Comment:
We shouldn't rename packages unless there is a conflict among imports. My suggestion earlier to not use `i` as a variable name was meant to steer you towards using something like `inst := f.NewInstance(...)`